src/pages/dar_application/SelectableDatasets.jsx

```diff
@@ -11,7 +11,8 @@ export default function SelectableDatasets(props) {
     // Populate parent state with the current state of datasets to be saved to the DAR
     const newSelectedDatasets = datasets.filter(ds => !removedIds.includes(ds.datasetId));
     setSelectedDatasets(newSelectedDatasets);
-  }, [removedIds, datasets, setSelectedDatasets]);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
We should be able to work through this lint warning - will need to dig through older PRs to see what approach we've taken.